fpmsyncd/routesync.cpp

```diff
@@ -106,6 +107,8 @@ enum {
     ROUTE_ENCAP_SRV6_ENCAP_SRC_ADDR    = 2,
 };
 
+#define MAX_MULTIPATH_NUM 256 //Same value used for FRR in SONiC
```

Comment:
We are planning to increase this to 514 based on recent requirements. Can you please initialize this to 514 instead? Here is the SONiC PR https://github.com/sonic-net/sonic-buildimage/pull/20745

Reply:
We are fixing this issue.